native_extensions/emoji_selector/src/lib.rs

```diff
@@ -128,6 +128,17 @@ fn render_selector<'a>(
             emoji_selector.focus();
         "#;
 
+    let emojis_to_display: Vec<(Group, Vec<&Emoji>)> = emojis::Group::iter()
+        .map(|group| {
+            let filtered_emojis: Vec<&Emoji> = group
+                .emojis()
+                .filter(|emoji| cfg!(target_os = "windows") && emoji.unicode_version().major() < 15)
```

Comment:
this `filter` call only returns `true` for the Windows OS. surely this isn't the desired behavior. 

https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.filter

Reply:
also, you may want to move the call to `filter` to within the `rsx!` macro further down, where you iterate over the emojis for a group.

Reply:
I see now that Darius found the bug already